### PR TITLE
examples: Add comments with sample run commands

### PR DESCRIPTION
**Pull request by CatalinStratu**
Hi @swinslow , I have added the necessary changes mentioned in #120

### CODE REVIEW

**Review by seabass-labrax — CHANGES_REQUESTED** [review]
Looks great, @CatalinStratu! I've left a few comments that probably ought to be fixed before this is merged. If you come up against issues when force-pushing to your fork (in order to update the pull request), please let me know and I'll be happy to help :)

**Comment by swinslow** [comment]
Hi @CatalinStratu, thanks for this! I haven't had a chance to review in detail yet (or to look at @seabass-labrax's feedback either), but one quick note:

Please take another look at my comment at https://github.com/spdx/tools-golang/pull/120#issuecomment-1079952170. I mentioned that the files to go into an `examples/sample-docs/` folder should just be the SPDX documents themselves, and not e.g. README files or code. It looks like this PR is still reinserting some of the README files from the spdx-examples repo.

Can you please take another look, and revise so that this is just adding the SPDX documents and nothing else for the `sample-docs/` folder?

**Comment by CatalinStratu** [comment]
> Hi @CatalinStratu, thanks for this! I haven't had a chance to review in detail yet (or to look at @seabass-labrax's feedback either), but one quick note:
> 
> Please take another look at my comment at [#120 (comment)](https://github.com/spdx/tools-golang/pull/120#issuecomment-1079952170). I mentioned that the files to go into an `examples/sample-docs/` folder should just be the SPDX documents themselves, and not e.g. README files or code. It looks like this PR is still reinserting some of the README files from the spdx-examples repo.
> 
> Can you please take another look, and revise so that this is just adding the SPDX documents and nothing else for the `sample-docs/` folder?

I made the necessary changes. Please review it.

File: examples/README.md [hunk 1]
```diff
@@ -43,7 +46,8 @@ Package and File license fields; and saving the resulting document to disk.
 This example demonstrates loading an SPDX tag-value file from disk into memory,
 generating a basic report listing counts of the concluded licenses for its
 files, and printing the report to standard output.
-
+#### Run project: * go run example_report.go ../sample-docs/example1/spdx/SPDXTagExample-v2.2.spdx
+*
```
[the remark this answers is not in the document]
Comment (CatalinStratu) [reply examples/README.md:50]: The README.md file no longer exists.

**Review by seabass-labrax — APPROVED** [review]
Excellent! Nice work, @CatalinStratu :smile:

**Comment by CatalinStratu** [comment]
> Excellent! Nice work, @CatalinStratu 😄

Thanks, @swinslow , what do you think?

File: examples/6-licensediff/example_licensediff.go [hunk 2]
```diff
@@ -8,7 +8,7 @@
 // This is generally only useful when run with two SPDX documents that
 // describe licenses for subsequent versions of the same set of files, AND if
 // they have the same identifier in both documents.
-
+// Run project: go run example_licensediff.go ../sample-docs/example1/spdx/example1.spdx ../sample-docs/example2/spdx/example2-src.spdx
```
[the remark this answers is not in the document]
Comment (swinslow) [reply examples/6-licensediff/example_licensediff.go]: This does appear to be a bug in either the tvloader parser or else possible the example file. I see that @CatalinStratu filed #124 so we can take a look there.

@CatalinStratu please note though that the files in example1/spdx/... and example2/spdx/... are not actually the same. They have the same filename, but take a look and you'll see that the contents are different. We can dig into this further in #124.

File: examples/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (swinslow) [reply examples/README.md:50]: @seabass-labrax is correct, this is in the main examples/README.md file so this does need a fix. I'll update separately after merging this PR.

**Comment by swinslow** [comment]
Thank you @CatalinStratu! There are a couple of subsequent tweaks but I can address them in a subsequent PR.

I'm going to squash these commits since it looks like the submitted ones have the extra README files getting added and then deleted.

**Pull request merged**